Tests/test_imagedraw.py

```diff
@@ -400,6 +400,28 @@ def test_floodfill_border(self):
         self.assert_image_equal(
             im, Image.open("Tests/images/imagedraw_floodfill2.png"))
 
+
+    @unittest.skipIf(hasattr(sys, 'pypy_version_info'),
```

Comment:
We shouldn't be skipping pypy, there's nothing here that should cause it to error out. If there is, it indicates a deeper problem somewhere.